xml/System/NullReferenceException.xml

```diff
@@ -151,6 +151,12 @@
   
      To address this issue, make sure that the argument passed to the method is not `null`, or handle the thrown exception in a `try…catch…finally` block. For more information, see [Exceptions](/dotnet/standard/exceptions/).  
   
+-   A <xref:System.NullReferenceException> exception can be thrown when a list is created without knowing the type, but the list was not initialized. The `GetList` method in the following example throws the exception at the line `emptyList.Add(value)`.
+  
+     :::code language="csharp" source="~/snippets/csharp/System/NullReferenceException/Overview/example3.cs" id="Snippet12":::
+  
+     To address this issue, make sure that the list is initialized (one way to do this is to call `Activator.CreateInstance` method instead of `FormatterServices.GetUninitializedObject`), or handle the thrown exception in a `try…catch…finally` block. For more information, see [Exceptions](/dotnet/standard/exceptions/).  
```

Comment:
```suggestion
     To address this issue, make sure that the list is initialized (one way to do this is to call `Activator.CreateInstance` instead of `FormatterServices.GetUninitializedObject`), or handle the thrown exception in a `try…catch…finally` block. For more information, see [Exceptions](/dotnet/standard/exceptions/).  
```